deployment_report/model/test/test_viya_deployment_report.py

```diff
@@ -632,3 +634,22 @@ def test_get_cadence_version_none() -> None:
     # make sure None is returned
     assert ViyaDeploymentReport().get_sas_component_resources(KubectlTest.Values.CADENCEINFO,
                                                               KubernetesResource.Kinds.CONFIGMAP) is None
+
+
+def test_get_db_info(report: ViyaDeploymentReport) -> None:
```

Comment:
Sorry, if get_db_info() and (get_cadence_version()) are moving to the utils files, the tests should probably move to test_viya_deployment_report_utils.py as well.

Reply:
code rearranged.